### ISSUE TITLE
Doing server side rendering

### ISSUE THREAD

**Issue by eXon** [parcel-bundler/parcel#71]
Parcel looks like the perfect bundler, I've been looking for a library just like this for a long time. The only missing piece for me is server-rendering. I would love to give a hand where needed because this project feels like a big improvement over the dreadful config I've been dealing with for a long time.

It looks like the bundle result is working like a charm with Node.js. The only problem I'm having is it doesn't look like I can export from the entry point.

I'm trying to export a function from the entry point so that I can require it within a Node.js server. However, it looks like nothing is exported. Is there a way to export from the bundle and have access at this export outside of the bundle?

Here is a repo with what I've been trying to achieve: https://github.com/eXon/parcel-test

Thank you for the amazing library 👍

**Comment by eXon** [comment]
Here is the gist of what I'm trying to do:

server-entry.js
```js
export default function middleware(req, res) {
 // ...
};
```

Bundling this with `parcel build entry-server.js --no-minify --out-dir dist-server --public-url dist`

server.js
```js
const middleware = require('./dist-server/entry-server.js').default;
// Expected middleware to be defined, but it isn't
```

**Comment by jakoblind** [comment]
Another way to do server side rendering is to run your backend code through babel and then import your React components straight from your backend ([better explanation](http://blog.jakoblind.no/getting-started-with-universal-rendering/)). Could that be an option for you?

**Comment by eXon** [comment]
@jakoblind I'm used to webpack and requireing image and assets with a hash name that would not be easily known just with babel, but it seems like parcel doesn't work that way. I've tried it and it seems to work pretty nicely :D

I'm also ignoring requires that are not useful in SSR with a babel plugin. It's just perfect. However, I'm blocked by #40 pretty badly. The client is crashing because of this. 

Also, I have a weird bug with the `--public-url` argument. It seems to be ignored. I run this `parcel build src/index.html --no-minify -d dist/client --public-url dist` and the path is /client instead of /dist in my index.html.

You can see it here: https://github.com/reactivestack/parcel-react-ssr

**Comment by chee** [comment]
@eXon that is fixed in parcel 1.0.3

here, i think: https://github.com/parcel-bundler/parcel/commit/bb7534945afe76c655901e4d51cab8ca8a7e7613#diff-ed26e1346331966874d8103cdcdbed47
